### PR TITLE
test: Add Proposal for multistaking module

### PR DESCRIPTION
**Pull request by GNaD13**
Fix proto
Add Proposal
- AddMultiStakingCoinProposal
- BondWeightProposal
- CLI
- Testing

### CODE REVIEW

File: x/multi-staking/keeper/proposal.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+func (k Keeper) AddMultiStakingCoinProposal(
+	ctx sdk.Context,
+	p *types.AddMultiStakingCoinProposal,
+) error {
+	_, found := k.GetBondWeight(ctx, p.Denom)
+	if found {
+		return fmt.Errorf("Error MultiStakingCoin %s already exist", p.Denom) //nolint:stylecheck
+	}
+
+	k.SetBondWeight(ctx, p.Denom, *p.BondWeight)
```
Comment (neitdung) [inline x/multi-staking/keeper/proposal.go:21]: We need to add a validation for checking `BondWeight` value is greater than zero.

File: x/multi-staking/keeper/proposal.go [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+func (k Keeper) BondWeightProposal(
+	ctx sdk.Context,
+	p *types.UpdateBondWeightProposal,
+) error {
+	_, found := k.GetBondWeight(ctx, p.Denom)
+	if !found {
+		return fmt.Errorf("Error MultiStakingCoin %s not found", p.Denom) //nolint:stylecheck
+	}
+
+	k.SetBondWeight(ctx, p.Denom, *p.UpdatedBondWeight)
```
Comment (neitdung) [inline x/multi-staking/keeper/proposal.go:42]: Add same validation at here

File: x/multi-staking/keeper/proposal_test.go [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+import (
+	"github.com/realio-tech/multi-staking-module/x/multi-staking/types"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	govv1types "github.com/cosmos/cosmos-sdk/x/gov/types/v1"
+)
+
+func (suite *KeeperTestSuite) TestAddHostZoneProposal() {
```
Comment (neitdung) [inline x/multi-staking/keeper/proposal_test.go:12]: Need to change test name

**Comment by neitdung** [comment]
LGTM

File: x/multi-staking/client/cli/proposal.go [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+		Short: "Submit an add multistaking coin proposal",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			from := clientCtx.GetFromAddress()
+			content := types.NewAddMultiStakingCoinProposal(
+				args[0], args[1], args[2], sdk.MustNewDecFromStr(args[3]),
```
[the remark this answers is not in the document]
Comment (catShaark) [reply x/multi-staking/client/cli/proposal.go:26]: you mean put all the args into a file ?

**Pull request merged**